python/src/TensorProductExperiment_doc.i.in

```diff
@@ -0,0 +1,134 @@
+%feature("docstring") OT::TensorProductExperiment
+"Tensor product experiment.
+
+Parameters
+----------
+experiments : list of :class:`~openturns.WeightedExperiment`
+    List of :math:`n` marginal experiments of the tensort product experiment. 
+    Each marginal experiment can have arbitrary dimension.
+
+Notes
+-----
+The tensor product design of experiments (DOE) is based on a collection of marginal multidimensional elementary designs of experiments. It is anisotropic in the sense that each marginal DOE does not necessarily have the same size. Using more points in one component may be useful if we want to get more accurate results in that particular direction. 
+
+Furthermore, each marginal DOE does not necessarily have a dimension equal to 1. In this sense, this is a generalization of a classical tensor product DOE. 
+
+Let :math:`\mathcal{X} \subset \mathbb{R}^{n_x}` be the integration domain and let :math:`g : \mathcal{X} \rightarrow \mathbb{R}^{n_y}` be an integrable function. Let :math:`f : \mathcal{X} \rightarrow \mathbb{R}` be a probability density function. The tensor product experiment produces an approximation of the integral :
+
+.. math::
+
+    \int_{\mathcal{X}} g(\boldsymbol{x}) f(\boldsymbol{x}) d\boldsymbol{x} \approx \sum_{i = 1}^{s_t} w_i g\left(\boldsymbol{x}_i\right)
+
+where :math:`s_t \in \mathbb{N}` is the size of the tensor product design of experiments, :math:`w_1, ..., w_{s_t} \in \mathbb{R}` are the weights and :math:`\boldsymbol{x}_1, ..., \boldsymbol{x}_{s_t} \in \mathbb{R}^{n_x}` are the nodes. 
+
+Let :math:`n_e \in \mathbb{N}` be the number of marginal elementary experiments. For each elementary experiment index :math:`j = 1, ..., n_e`, let :math:`s_j \in \mathbb{N}` be its size, representing the number of nodes in the marginal elementary quadrature. For any index :math:`j = 1, ..., n_e`, let :math:`d_j \in \mathbb{N}` be the dimension of the :math:`j`-th experiment. Let :math:`\left\{ Q_{s_j}^{(d_j)} \right\}_{j = 1, ..., n_e}` be the collection of marginal elementary experiments.
```

Comment:
Lines are way too long, this is not easy to review.

Reply:
Ok. I cut them approximately to the 80 column and commit that. It is, however, much easier to read that particular content at: https://8743-40717169-gh.circle-artifacts.com/0/html/user_manual/_generated/openturns.TensorProductExperiment.html